scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/service/ScheduleManagerTest.java

```diff
@@ -46,7 +50,7 @@
 @ActiveProfiles("ScheduleDaoMock")
 @RunWith(SpringRunner.class)
 @SpringBootTest
-public class ScheduleManagerTest  extends TestConfiguration {
```

Comment:
testDeleteSchedules, has the test for delete active schedules
Mockito.verify(activeScheduleDao, Mockito.times(1)).findAllActiveSchedulesByAppId(appId);
Mockito.verify(activeScheduleDao, Mockito.times(8)).delete(1L);

Which will now change because of above change.